build/media-suite_helper.sh

```diff
@@ -1743,6 +1743,27 @@ EOF
             cat > "$LOCALDESTDIR"/bin/ab-pkg-config-static.bat
 }
 
+create_cmake_toolchain() {
+    local _win_path_LOCALDESTDIR="$(cygpath -m $LOCALDESTDIR)"
+    local _win_path_MINGW_PREFIX="$(cygpath -m $MINGW_PREFIX)"
+    local toolchain_file=(
+        "SET(CMAKE_C_COMPILER gcc)"
```

Comment:
Is it necessary to hardcode gcc? Maybe better to let cmake detect c compiler/use CC variable

Reply:
Sure, those three lines were sort of copied from https://github.com/shinchiro/mpv-winbuild-cmake/blob/master/toolchain.cmake.in

Reply:
I see, I see